server/db.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+const url = 'mongodb://localhost/products';
+
+mongoose.connect(url, { useNewUrlParser: true }, err => {
+  if (err) throw new Error(err);
```

Comment:
`throw new Error(err)` will produce an unhandled promise because there's no try / catch around
terminal is warning about that.

Reply:
good catch ;) i'll deal with it. thx